Source/Data/BPMFMappings.txt

```diff
@@ -87070,6 +87082,7 @@
 珍愛 ㄓㄣ ㄞˋ
 珍本 ㄓㄣ ㄅㄣˇ
 珍物 ㄓㄣ ㄨˋ
+珍玩 ㄓㄣ ㄨㄢˊ
```

Comment:
We can add this given https://dict.concised.moe.edu.tw/dictView.jsp?ID=29947#searchL and https://dict.revised.moe.edu.tw/dictView.jsp?ID=116352&word=%E7%8F%8D%E7%8E%A9#searchL